Build/Customizations/releaseProcessScript.config

```diff
@@ -2,7 +2,7 @@
 <settings>
   <jira>
     <jiraUrl>https://re-motion.atlassian.net</jiraUrl>
-    <jiraProjectKey>RMSRCBUILD</jiraProjectKey>
+    <jiraProjectKey>DEFINETLYNOTAPROJECTKEY</jiraProjectKey>
```

Comment:
Now that we made sure that the tests do not use the actual jira instance but mocks is there any reason to keep this change?

Reply:
I think the better question would be whether we should undo the change as it gives safety against jira dying if someone creates a new test and somehow forgets to check the config that Jira doesnt die. 

So unless we want RM in our test config, I think it would benefit us to have the extra safety.